packages/api/src/utils/billing.js

```diff
@@ -167,14 +167,14 @@ export function createMockUserCustomerService () {
   /**
    * @returns {Promise<void>}
    */
-  const upsertUserCustomer = async (userId, customerId) => {
+  const insertUserCustomer = async (userId, customerId) => {
     userIdToCustomerId.set(userId, customerId)
```

Comment:
you might want to update this to throw on duplicates like the postgrest one will do